bundles/org.openhab.binding.enphase/pom.xml

```diff
@@ -14,4 +14,17 @@
 
   <name>openHAB Add-ons :: Bundles :: Enphase Binding</name>
 
+  <properties>
+    <jsoup.version>1.15.3</jsoup.version>
```

Comment:
I picked the most used version among other addons. Because this dependency is installed as feature it means each different version will be installed and could cause problems with other bindings if something changed and the addon didn't expect it (It's already an issue because multiple addons have different versions). Ideally all addons should probably use the same version, and update them at once. But I would guess that is a bit outside of the scope of this pull request.